src/pages/es/guides/images.md

```diff
@@ -104,29 +116,51 @@ La [carpeta `public/`](/es/core-concepts/project-structure/#public) es para arch
 
 ## Integración Image de Astro
 
-:::caution
-Cuando instales la integración `@astrojs/image`, los archivos `.astro` no podrán usar etiquetas estándar de HTML `<img>` para imágenes locales en la carpeta `src` de tu proyecto. Todas las imágenes locales deben usar los componentes de la integración en lugar de etiquetas estándar de HTML.
+La integración oficial de Astro, imágenes en Astro proporciona dos componentes diferentes Astro para renderizar imágenes optimizadas: `<Image />` y `<Picture />`. Tiene soporte para todos los sitios estáticos y para [algunos hosts de server-side-rendering](es/guides/integrations-guides/image/#instalación)
```

Comment:
Ah, because this page isn’t translated, we do still need `#installation` rather than `#instalación`:

```suggestion
La integración oficial de Astro, imágenes en Astro proporciona dos componentes diferentes Astro para renderizar imágenes optimizadas: `<Image />` y `<Picture />`. Tiene soporte para todos los sitios estáticos y para [algunos hosts de server-side-rendering](/es/guides/integrations-guide/image/#installation)
```